ca/certificates.go

```diff
@@ -566,16 +568,16 @@ func GetRemoteSignedCertificate(ctx context.Context, csr []byte, token string, r
 	caClient := api.NewNodeCAClient(conn)
 
 	// Send the Request and retrieve the request token
-	issueRequest := &api.IssueNodeCertificateRequest{CSR: csr, Token: token}
+	issueRequest := &api.IssueNodeCertificateRequest{CSR: csr, Token: config.Token}
 	issueResponse, err := caClient.IssueNodeCertificate(ctx, issueRequest)
 	if err != nil {
-		r.Observe(peer, -remotes.DefaultObservationWeight)
+		config.Remotes.Observe(peer, -remotes.DefaultObservationWeight)
 		return nil, err
 	}
 
 	// Send back the NodeID on the nodeInfo, so the caller can know what ID was assigned by the CA
-	if nodeInfo != nil {
-		nodeInfo <- *issueResponse
+	if config.NodeInfo != nil {
```

Comment:
Yeah, was thinking about removing it. I had a sense that the pending stuff was still wired up but we just weren't using it on the CA side. Looking closer, it seems like this has atrophied beyond the point where it's useful. All the channel does is set `n.nodeMembership`, which is exposed through a `NodeMembership` method. Nothing in either swarmkit or docker calls this method.

I'll remove the channel and the associated approval stuff.